spec/reek/smells/attribute_spec.rb

```diff
@@ -127,5 +127,15 @@ class OtherKlass
       '
       expect(src).to reek_of(:Attribute)
     end
+
+    it 'can be suppressed with a preceding code comment' do
```

Comment:
Can we make this into a "context" and add a safety test here for the dame source but without the comment? You know, "test your tests", at least when its not obvious :)